src/includes/getting-started-config/angular.mdx

```diff
@@ -0,0 +1,100 @@
+This snippet includes our [performance monitoring](/product/performance/) package, which registers and configures the Tracing integration, including custom Angular routing instrumentation:
+
+```javascript
+import { enableProdMode } from "@angular/core";
+import { platformBrowserDynamic } from "@angular/platform-browser-dynamic";
+import * as Sentry from "@sentry/angular";
+import { Integrations } from "@sentry/tracing";
+import { AppModule } from "./app/app.module";
+Sentry.init({
+  dsn: "___PUBLIC_DSN___" ,
+  integrations: [
+    new Integrations.BrowserTracing({
+      tracingOrigins: ["localhost", "https://yourserver.io/api"],
+      routingInstrumentation: Sentry.routingInstrumentation,
+    }),
+  ],
+  tracesSampleRate: 1.0, // Be sure to lower this in production
```

Comment:
fyi this is broken syntax here - im not sure what its supposed to be just know this isnt valid JS.

Reply:
I'm sure it's not right :) I was working on the concept of the include, thus it needs a technical review prior to merging.

Reply:
Fixed in technical review.